Hello, I think this change broke python 3.8 support:

Hello, I think this change broke python 3.8 support:

```
../lib/python3.8/site-packages/yarl/_url.py:606: in URL
    def query(self) -> MultiDictProxy[str]:
E   TypeError: 'type' object is not subscriptable
```
If I downgrade to release 1.9.7 it works again.

You might want to update python requirements to indicate python>=3.9 is required

_Originally posted by @sergeniushpe in https://github.com/aio-libs/yarl/issues/1084#issuecomment-2330345023_

--- a/CHANGES/1084.bugfix.rst
+++ b/CHANGES/1084.bugfix.rst
@@ -1,0 +1,1 @@
+1107.bugfix.rst

--- a/CHANGES/1105.bugfix.rst
+++ b/CHANGES/1105.bugfix.rst
@@ -1,0 +1,1 @@
+1107.bugfix.rst

--- a/CHANGES/1107.bugfix.rst
+++ b/CHANGES/1107.bugfix.rst
@@ -1,3 +1,1 @@
-Fixed compatibility with ``multidict`` versions where ``MultiDictProxy`` is not a ``Generic`` -- by :user:`bdraco`.
-
-A :exc:`TypeError` would be raised in this case.
+Fixed a :exc:`TypeError` with ``MultiDictProxy`` and Python 3.8 -- by :user:`bdraco`.

--- a/CHANGES/1107.bugfix.rst
+++ b/CHANGES/1107.bugfix.rst
@@ -1,0 +1,3 @@
+Fixed compatibility with ``multidict`` versions where ``MultiDictProxy`` is not a ``Generic`` -- by :user:`bdraco`.
+
+A :exc:`TypeError` would be raised in this case.

--- a/yarl/_url.py
+++ b/yarl/_url.py
@@ -640,7 +640,7 @@ class URL:
         return self._PATH_UNQUOTER(self.raw_path)
 
     @cached_property
-    def query(self) -> MultiDictProxy[str]:
+    def query(self) -> "MultiDictProxy[str]":
         """A MultiDictProxy representing parsed query parameters in decoded
         representation.
 

